src/palette/internal/palettelayout.cpp

```diff
@@ -92,6 +215,168 @@ void PaletteLayout::layout(Articulation* item, const Context&)
     item->setbbox(bbox.translated(-0.5 * bbox.width(), 0.0));
 }
 
+void PaletteLayout::layout(BagpipeEmbellishment* item, const Context& ctx)
+{
+    SymId headsym = SymId::noteheadBlack;
+    SymId flagsym = SymId::flag32ndUp;
+
+    noteList nl = item->getNoteList();
```

Comment:
It would be great to rename this to NoteList

Reply:
It's not just a getter, so I decided to keep it like that